packs/feats/energized-spark.json

```diff
@@ -234,12 +234,34 @@
                 "mode": "override",
                 "predicate": [
                     "damage:type:spirit",
-                    "item:trait:exemplar",
-                    "energized-spark-damage"
+                    "energized-spark-damage",
+                    {
+                        "or": [
+                            {
+                                "and": [
+                                    "item:type:feat",
+                                    "item:trait:exemplar"
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "item:type:action",
+                                    "item:trait:transcendence"
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "action:strike",
+                                    "damage:tag:exemplar-damage"
```

Comment:
_chews_ needs more `damage`

Reply:
does look redundant now that you mention it huh